specification/durabletask/DurableTask.Management/tspconfig.yaml

```diff
@@ -0,0 +1,17 @@
+parameters:
+  "service-dir":
+    default: ""
+  "dependencies":
+    "additionalDirectories": []
+    default: ""
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
```

Comment:
Please add the SDK emitters options accordingly in order to generate SDK based on TypeSpecs.

Reply:
@raych1 I added them. Let me know if this is correct - I couldn't find the right documentation on this, so looked at other providers and what they have done.

Reply:
@weidongxu-microsoft @chunyu3 @qiaozha @tadelesh @msyyc 
please review the emitter options.

Reply:
Looks good for Python.

Reply:
@raych1 

Are we allowed to enhance this doc? https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/Getting-started-with-TypeSpec-specifications.md#arm
(especially this https://github.com/Azure/typespec-azure/blob/main/eng/feeds/arm/tspconfig.yaml)

I can make a PR, if it is OK for us to update.

Reply:
@weidongxu-microsoft 
That's document at the typespec-azure repo which will need to update I agree, feel free to update. But I will suggest user to refer to sample project for the emitter configurations. https://github.com/Azure/azure-rest-api-specs/blob/main/specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml

Reply:
@raych1 the config here is updated to match the example at https://github.com/Azure/azure-rest-api-specs/blob/main/specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml